mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -0,0 +1,202 @@
+use std::path::Path;
+use std::vec::Vec;
+
+use config::{Config, ConfigError, Environment, File, FileFormat};
+use serde::Deserialize;
+
+pub const DEFAULTS: &str = include_str!("../config/default.json");
+pub const ENVIRONMENT_PREFIX: &str = "iotedge";
+
+#[derive(Debug, Clone, Default, PartialEq, Deserialize)]
+pub struct Settings {
+    #[serde(flatten)]
+    nested_bridge: NestedBridgeSettings,
+
+    #[serde(flatten)]
+    subscriptions: Subscriptions,
+
+    #[serde(flatten)]
+    forwards: Forwards,
+}
+
+impl Settings {
+    pub fn new() -> Result<Self, ConfigError> {
+        let mut config = Config::new();
+
+        config.merge(File::from_str(DEFAULTS, FileFormat::Json))?;
+        config.merge(Environment::with_prefix(ENVIRONMENT_PREFIX))?;
```

Comment:
Is the bridge configured from the same config file than the broker, or different?

Reply:
I think it is different. It is also tbd if it is from a file or using a special sub for receiving config